### PR TITLE
feature/added-index.html

### PR DESCRIPTION
**Pull request by ikeman09**

### CODE REVIEW

**Comment by JohnServo24** [comment]
adasdasdasd

**Comment by ikeman09** [comment]
BANGAA MUCODE OY

**Comment by abbypandaan** [comment]
hiiii

**Comment by RoldValiente** [comment]
LetLeniLead

File: index.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+<p>HELLO HAROLD I LOVE U</p>
```
Comment (RoldValiente) [inline index.html:3]: No thanks

File: index.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<html>
```
Comment (JohnServo24) [inline index.html:1]: <html>

**Pull request merged**